lib/bootloader_setup.pm

```diff
@@ -894,7 +899,12 @@ sub tianocore_enter_menu {
 }
 
 sub tianocore_disable_secureboot {
-    my $basetest = shift;
+
+    my ($basetest, $revert) = @_;
+
+    my $neelle_sb_conf_attempt = $revert ? 'tianocore-devicemanager-sb-conf-disabled' : 'tianocore-devicemanager-sb-conf-attempt-sb';
```

Comment:
I do not quite understand the meaning about **neelle**, is it the term from tianocore-devicemanager?

Reply:
Yes, you are correct,  we can enable or disable secureboot support upon this matched needle

Reply:
good to know, thanks.